### DISCUSSION TITLE
Stop code execution

### DISCUSSION THREAD

**Discussion by rohan-uiuc** [ricklamers/gpt-code-ui#8] [Q&A]
Is there a way to stop the code execution(not generation) apart from restarting the kernel?

Generated code might have loops/recursions that lead to infinite execution. There should be a way to stop the execution of code apart from stopping the entire application or restarting the kernel. Perharps, a stop button in the chat window. 

Thanks!

**Comment by ricklamers** [comment]
This would be a good feature for someone to code. Added it as a Good First issue https://github.com/ricklamers/gpt-code-ui/issues/9
